docker/dockerfile.merlin

```diff
@@ -252,13 +280,21 @@ ENV JUPYTER_CONFIG_DIR=/tmp/.jupyter
 ENV JUPYTER_DATA_DIR=/tmp/.jupyter
 ENV JUPYTER_RUNTIME_DIR=/tmp/.jupyter
 
-ARG MERLIN_VER
-ARG CORE_VER
-ARG MODELS_VER
-ARG NVTAB_VER
-ARG SYSTEMS_VER
-ARG TF4REC_VER
-ARG DL_VER
+ARG MERLIN_VER=main
+ARG CORE_VER=main
+ARG MODELS_VER=main
+ARG NVTAB_VER=main
+ARG SYSTEMS_VER=main
+ARG TF4REC_VER=main
+ARG DL_VER=main
+
+ENV MERLIN_VER=${MERLIN_VER}
```

Comment:
What is the reason for storing these values in the container? We use them for pulling in the code in the following instructions. Are we changing something that will leverage this?

Reply:
They were already stored in the container at the top of the Dockerfile on Lines 14-22.  They needed to be replicated here (well, somewhere after the `FROM ${BASE_IMAGE} as base` command) so the subsequent commands like `git clone --branch ${MERLIN_VER}` would work.  Without specifying `=main`, the default is empty and that `git clone` command fails when building via commands like `docker build - < docker/dockerfile.merlin` where the argument value isn't explicitly set.